README.md

```diff
@@ -106,7 +108,15 @@ concept behind the implementation
 
 Once you've successfully made some changes, built, ran, tested and pushed to your GitHub remote - then you should submit a pull request for the changes to be applied on the upstream repo.
 
-Use [GitHub's interface](https://help.github.com/articles/about-pull-requests/) to create new pull requests. Once submitted, the code will go through some automated tests, integration and peer-reviews.
+To create new pull requests:
+
+* Use [GitHub's interface](https://help.github.com/articles/about-pull-requests/)
+* Use [hub](https://hub.github.com/) command.
+  ```sh
+  $ hub pull-request -b Tendrl/ui:master -h <Github-username>:<your-branch>
```

Comment:
I doubt if the base is `Tendrl/ui:master` or `Tendrl:master`. Please suggest.

Reply:
should be the latter.

Reply:
@sankarshanmukhopadhyay Updated. Thanks.